src/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+    <title>To Do Tasks</title>
+</head>
+<body>
+    <div class="to-do-container">
+        <div class="row">
+            <div class="head-col"><h2> Today's To Do</h2></div>
+            <div class="head-col"><span><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="#c4c5c6" class="bi bi-arrow-repeat" viewBox="0 0 16 16">
+                <path d="M11.534 7h3.932a.25.25 0 0 1 .192.41l-1.966 2.36a.25.25 0 0 1-.384 0l-1.966-2.36a.25.25 0 0 1 .192-.41zm-11 2h3.932a.25.25 0 0 0 .192-.41L2.692 6.23a.25.25 0 0 0-.384 0L.342 8.59A.25.25 0 0 0 .534 9z"/>
+                <path fill-rule="evenodd" d="M8 3c-1.552 0-2.94.707-3.857 1.818a.5.5 0 1 1-.771-.636A6.002 6.002 0 0 1 13.917 7H12.9A5.002 5.002 0 0 0 8 3zM3.1 9a5.002 5.002 0 0 0 8.757 2.182.5.5 0 1 1 .771.636A6.002 6.002 0 0 1 2.083 9H3.1z"/>
+              </svg></span> </div>
+        </div>        
+        <div class="row">
+            <div class="col">
+                <form>
+                    <input type="text" id="new-item" placeholder="Add to your list..." aria-label="Add a Task" />
+                </form>
+            </div>
+            <div class="col"><span><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="#c4c5c6" class="bi bi-arrow-return-left" viewBox="0 0 16 16">
+                <path fill-rule="evenodd" d="M14.5 1.5a.5.5 0 0 1 .5.5v4.8a2.5 2.5 0 0 1-2.5 2.5H2.707l3.347 3.346a.5.5 0 0 1-.708.708l-4.2-4.2a.5.5 0 0 1 0-.708l4-4a.5.5 0 1 1 .708.708L2.707 8.3H12.5A1.5 1.5 0 0 0 14 6.8V2a.5.5 0 0 1 .5-.5z"/>
+              </svg></span></div>
+        </div>
+            <ul class="container">
+                <!-- <li class="draggable" draggable="true"> 
+                    <input class="toggle" type="checkbox" aria-label="Task Complete"/>
+                    <textarea name="task"></textarea>
+                    <span ><svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" fill="currentColor" class="bi bi-three-dots-vertical" viewBox="0 0 16 16">
+                        <path d="M9.5 13a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0zm0-5a1.5 1.5 0 1 1-3 0 1.5 1.5 0 0 1 3 0z"/>
+                      </svg></span>
+                </li> -->
+                <!-- For Dynamic Lists -->
```

Comment:
- [OPTIONAL] It will be nice to remove commented code from your code base, as this makes your code base dirty and less readable.